core/storage/app_feedback_report/gae_models.py

```diff
@@ -537,11 +551,16 @@ class in the required format with the arguments provided.
 
         Returns:
             str. The generated ID for this entity of the form
-            '[platform]:[ticket_id]:[date.seconds]'.
+            '[platform]:[ticket_id]:[stats_date_msec]'.
```

Comment:
Date is different from datetime. In this docstring you say date; in the implementation it looks like datetime. Please standardize.

IMO a format like YYYY-MM-DD is suitable for date; it's unambiguous. Having a date at millisec resolution does not really make sense.

Reply:
Ack, I will use the YYYY-MM-DD date for this